data/json/items/gun/shot.json

```diff
@@ -1,5 +1,32 @@
 [
   {
+    "id": "abzats",
+    "copy-from": "m2browning",
+    "type": "GUN",
+    "name": "heavy automatic shotgun",
+    "description": "Modified from the massive M2 Browning, this heavy machine gun has been re-chambered and re-bored for shotgun shells, and completely redesigned to be wielded unmounted by one user.",
+    "material": [ "steel", "wood" ],
+    "ammo": "shot",
+    "skill": "shotgun",
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "AUTO", "auto", 5 ]
+    ],
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "mechanism", 4 ], [ "rail", 1 ] ],
+    "magazines": [ [ "shot", [ "beltshot" ] ] ],
+    "relative": {
+      "weight": -16000,
```

Comment:
It's supposed to be hugely impractical! If you want a rambo gun it should be rambo's only gun and require a character like him to make use of it. You shouldn't drop 16kg of weight here